packages/grafana-data/src/transformations/transformers/calculateField.test.ts

```diff
@@ -96,9 +96,9 @@ describe('calculateField transformer w/ timeseries', () => {
       options: {
         mode: CalculateFieldMode.ReduceRow,
         reduce: {
-          include: 'B',
+          include: ['B'],
           reducer: ReducerID.mean,
-        },
+        } as ReduceOptions,
```

Comment:
nit: if you want to escape the strict null error trap 😉 
```suggestion
        } as unknown as ReduceOptions,
```